engine-standalone-tracing/src/types/mod.rs

```diff
@@ -24,7 +24,9 @@ impl Depth {
     }
 
     pub fn decrement(&mut self) {
-        self.0 -= 1;
+        if !self.is_zero() {
```

Comment:
Could use `saturating_sub` instead. This is logically equivalent to the current implementation but without the `if` so it is a little less code and a little more efficient at runtime.

Reply:
Actually I was thinking the same way. Thanks